tools/releaseBuild/azureDevOps/vpackRelease.yml

```diff
@@ -0,0 +1,98 @@
+name: vpack-$(Build.BuildId)
+trigger:
+  branches:
+    include:
+    - master
+    - release*
+pr:
+  branches:
+    include:
+    - master
+    - release*
+
+variables:
+  - name: DOTNET_CLI_TELEMETRY_OPTOUT
+    value: 1
+  - name: POWERSHELL_TELEMETRY_OPTOUT
+    value: 1
+  - group: Azure Blob variable group
+
+# Set AzDevOps Agent to clean the machine after the end of the build
+resources:
+- repo: self
+  clean: true
+
+jobs:
+- job: vpack
+  displayName: vpack
+  condition: succeeded()
+  pool: Package ES Standard Build
+  steps:
+
+  - template: ./templates/SetVersionVariables.yml
+    parameters:
+      ReleaseTagVar: $(ReleaseTagVar)
+
+  - powershell:  Write-Host "##vso[build.updatebuildnumber]$env:BUILD_SOURCEBRANCHNAME-$env:BUILD_SOURCEVERSION-$((get-date).ToString("yyyyMMddhhss"))"
+    displayName: Set Build Name for Non-PR
+    condition: ne(variables['Build.Reason'], 'PullRequest')
+
+  - pwsh: |
+      $azcopy = "C:\Program Files (x86)\Microsoft SDKs\Azure\AzCopy\AzCopy.exe"
+
+      & $azcopy /Source:https://$(StorageAccount).blob.core.windows.net/$(AzureVersion) /Dest:$(System.ArtifactsDirectory) /S /SourceKey:$(StorageAccountKey)
+    displayName: 'Download Azure Artifacts'
+
+  - pwsh: 'Get-ChildItem $(System.ArtifactsDirectory)\* -recurse | Select-Object -ExpandProperty Name'
+    displayName: 'Capture Artifact Listing'
+
+  - pwsh: |
+      if('$(ReleaseTagVar)' -like '*-*')
+      {
+          throw "Don't release a vpack for a preview"
```

Comment:
The rc releases are expected to be "go live" so should be make an exception for those?